internal/persistence/sql/traverser.go

```diff
@@ -145,7 +145,7 @@ func (t *Traverser) TraverseSubjectSetRewrite(ctx context.Context, start *relati
 	}
 
 	if len(relations) > 0 {
-		_, span := t.d.Tracer(ctx).Tracer().Start(ctx, "persistence.sql.TraverseSubjectSetRewrite")
+		ctx, span := t.d.Tracer(ctx).Tracer().Start(ctx, "persistence.sql.TraverseSubjectSetRewrite")
```

Comment:
```suggestion
		var span *otel.Span
		ctx, span = t.d.Tracer(ctx).Tracer().Start(ctx, "persistence.sql.TraverseSubjectSetRewrite")
```
because of scoping?

Reply:
Would not make a difference right now, but might later on. Or maybe just move that to the beginning of the function?

Reply:
Actually you're correct! fixed, thx